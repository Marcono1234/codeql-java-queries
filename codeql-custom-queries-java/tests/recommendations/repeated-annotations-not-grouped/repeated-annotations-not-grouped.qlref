./recommendations/INCOMPLETE_repeated-annotations-not-grouped.ql
