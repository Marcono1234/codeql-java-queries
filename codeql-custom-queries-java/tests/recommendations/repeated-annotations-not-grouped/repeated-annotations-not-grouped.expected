| Test.java:19:5:19:10 | Other | Annotation is between $@ and $@ annotation of the same type | Test.class:0:0:0:0 | Marker | this | Test.class:0:0:0:0 | Marker | this |
| Test.java:23:13:23:18 | Other | Annotation is between $@ and $@ annotation of the same type | Test.class:0:0:0:0 | Marker | this | Test.class:0:0:0:0 | Marker | this |
| Test.java:29:5:29:10 | Other | Annotation is between $@ and $@ annotation of the same type | Test.class:0:0:0:0 | Marker | this | Test.class:0:0:0:0 | Marker | this |
| Test.java:32:5:32:10 | Other | Annotation is between $@ and $@ annotation of the same type | Test.class:0:0:0:0 | Marker | this | Test.class:0:0:0:0 | Marker | this |
